.env.example

```
@@ -10,6 +10,7 @@ OPENAI_API_KEY=
 DATABASE_URL=               # auto-generated by setup:local-db
 
 # ── Optional ──────────────────────────────────
+OPENAI_ADMIN_API_KEY=       # enables server-side OpenAI usage/cost telemetry in backoffice
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)